pymtl3/stdlib/rtl/valrdy_queues.py

```diff
@@ -75,7 +75,7 @@ def up_bypq_set_enq_rdy():
 
       @s.update
       def up_bypq_internal():
-        s.buf.en    = (not s.deq.rdy) & (s.enq.val & s.enq.rdy)
+        s.buffer.en    = (not s.deq.rdy) & (s.enq.val & s.enq.rdy)
```

Comment:
It might be good to add some test cases to catch SV keywords like this?

Reply:
Yes definitely. The plan is to have another PR focusing on different aspects of the translation/import passes.